src/GameSrc/gameobj.c

```diff
@@ -1193,9 +1193,10 @@ void show_obj(ObjID cobjid) {
                             (objCritters[_fr_cobj->specID].mood == AI_MOOD_ATTACKING)) {
                             uchar c = CyberCritterProps[SCNUM(cobjid)].alt_vcolors[foog], nc;
                             int hp_state;
+                            int denom = ObjProps[objtrip].hit_points; if (denom == 0) denom = 1;
```

Comment:
Could we move that `if` statement to its own line? Would be a bit clearer that way.